### DISCUSSION TITLE
Add a rule considersing all the edges

### DISCUSSION THREAD

**Discussion by KaminiAggarwal** [lab-v2/pyreason#49] [Q&A]
Hi, 

I wanted some help regarding creating a rule that considers all the edges of a node. 

For example, considering below tutorial, if I have to create a rule like:
- A customer is popular only if all the friends of a customer are popular.

https://pyreason-rtd.readthedocs.io/en/latest/tutorials/Advanced%20tutorial.html

**Comment by dyumanaditya** [comment]
Hello,

Unfortunately this behaviour is not supported yet. We plan to build support in future releases

**Reply by f-bdolan** [reply]
Do we have an estimate on when this might be picked up or guidance on how can contribute.

**Reply by dyumanaditya** [reply]
Replied in #50 [here](https://github.com/lab-v2/pyreason/issues/50#issuecomment-2096038303)
